### ISSUE TITLE
Feature Request: giving reasoning for marking

### ISSUE THREAD

**Issue by LuminatiHD** [shinigami-eyes/shinigami-eyes#105]
I think itd be cool if there was like a little field you had to/could fill out for a reason for the marking. Like, putting a link to a post were the person in question said something trans positive/transphobic.

**Comment by Seff27** [comment]
Yeah, I agree with that. There's a YouTuber my GF watches, Deadwing Dork, and neither of us really think he's at all transphobic, so it'd be nice to at least know why he was marked.

**Comment by Ralimbahere** [comment]
Yeah People on Lemmy critiqued the extension because of that.

**Comment by thecaralice** [comment]
Yeah I would like to know why https://en.wikipedia.org/wiki/Zero is marked as trans-friendly
